smrt/interface/coherent_flat.py

```diff
@@ -104,7 +111,7 @@ def coherent_transmission_matrix(self, frequency, eps_1, eps_2, mu1, npol):
             npol: Number of polarization.
 
         Returns:
-            The transmission matrix.
+            The transmission matrix, :py:mod:`smrt.core.fresnel.fresnel_transmission_matrix`.
```

Comment:
please remove :py:mod:`smrt.core.fresnel.fresnel_transmission_matrix`, this is not exactly what this function is doing